go.mod

```diff
@@ -4,6 +4,6 @@ go 1.22.0
 
 require (
 	github.com/bwesterb/go-ristretto v1.2.3
-	golang.org/x/crypto v0.11.1-0.20230711161743-2e82bdd1719d
-	golang.org/x/sys v0.10.0
+	golang.org/x/crypto v0.31.0
```

Comment:
The current Go 1.24 development version uses v0.30.0: https://github.com/golang/go/blob/9118060040ab7af4d384c2c1e2862673e2bbb8c3/src/go.mod
(It uses the same x/sys version, v0.28.0)

The only difference with v0.31.0 is a fix for a SSH vulnerability which is irrelevant to CIRCL: https://github.com/golang/crypto/compare/v0.30.0...v0.31.0

If we are going to update this, I would suggest sticking to `v0.30.0`. Otherwise we would have to modify `go.mod`, `go.sum` and `vendor/modules.txt` when vendoring circl into stdlib. Doing so could cause merge conflicts in future rebases.